src/main/java/net/dries007/tfc/common/blocks/wood/TFCTrappedChestBlock.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the EUPL, Version 1.2.
+ * You may obtain a copy of the Licence at:
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ */
+
+package net.dries007.tfc.common.blocks.wood;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.stats.Stat;
+import net.minecraft.stats.Stats;
+import net.minecraft.util.Mth;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.block.entity.ChestBlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+
+import net.dries007.tfc.common.blockentities.TFCBlockEntities;
+import net.dries007.tfc.common.blocks.ExtendedProperties;
+
+public class TFCTrappedChestBlock extends TFCChestBlock
+{
+    public TFCTrappedChestBlock(ExtendedProperties properties, String textureLocation)
+    {
+        super(properties, textureLocation, TFCBlockEntities.TRAPPED_CHEST::get);
```

Comment:
Same as above, `TFCBlockEntities.TRAPPED_CHEST` should be valid to pass through.